doc/source/links.rst

```diff
@@ -89,8 +89,9 @@
 .. _course_intro_python: https://courses.ansys.com/index.php/courses/intro-to-python/
 .. _course_getting_started_pymapdl: https://courses.ansys.com/index.php/courses/getting-started-with-pymapdl/
 .. _course_intro_apdl: https://courses.ansys.com/index.php/courses/intro-to-ansys-mechanical-apdl-scripting/
+.. _course_fem_track: https://courses.ansys.com/index.php/learning-track/introduction-to-finite-element-methods/
 
-.. #External links
+.. #External linksm
```

Comment:
I assume the added "m" is intentional? Asking only because it looks weird.

Reply:
it is not xD